src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -534,6 +533,12 @@ public void drawDamagePreview(Graphics g, DamagePopup data)
     int estimateX = (data.coords.xCoord * tileSize) + (tileSize / 2);
     int estimateY = Math.max((data.coords.yCoord * tileSize) - dmgImage.getHeight() / 2, dmgImage.getHeight() / 2); // Don't want it floating off-screen
     SpriteUIUtils.drawImageCenteredOnPoint(g, dmgImage, estimateX, estimateY);
+
+    if( spaceEmpty && 0 != data.coords.yCoord )
+    {
+      int arrowY = (data.coords.yCoord * tileSize) + (tileSize / 2);
```

Comment:
```suggestion
      int arrowY = (data.coords.yCoord * tileSize) + (tileSize / 2) - 1;
```

Reply:
I'm assuming this is to break the bottom border of the preview like in your mockup?